databases/week2/creatingDatabase/week2HWscript.sql

```diff
@@ -0,0 +1,102 @@
+ -- FOREIGN KEY (brandID) REFERENCES brand(brandID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `city` (
+  `cityID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `cityName` varchar(255) NOT NULL,
+  `countryID` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`cityID`),
+  CONSTRAINT `city_countryID` FOREIGN KEY (`countryID`) REFERENCES `country` (`countryID`) ON DELETE CASCADE ON UPDATE CASCADE
+  -- FOREIGN KEY (countryID) REFERENCES country(countryID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `store` (
+  `storeID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `storeName` varchar(255) NOT NULL,
+  `cityID` int(10) unsigned NOT NULL,
+  `countryID` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`storeID`),
+  CONSTRAINT `store_countryID` FOREIGN KEY (`countryID`) REFERENCES `country` (`countryID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `store_cityID` FOREIGN KEY (`cityID`) REFERENCES `city` (`cityID`) ON DELETE CASCADE ON UPDATE CASCADE
+ -- FOREIGN KEY (countryID) REFERENCES country(countryID),
+ -- FOREIGN KEY (cityID) REFERENCES city(cityID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `order_item` (
+  `orderID` int(10) unsigned NOT NULL,
+  `itemID` int(10) unsigned NOT NULL,
+  `itemQuantity` int(10) NOT NULL,
+  PRIMARY KEY (`orderID`, `itemID`),
+  CONSTRAINT `order_item_orderID` FOREIGN KEY (`orderID`) REFERENCES `orders` (`orderID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `order_item_itemID` FOREIGN KEY (`itemID`) REFERENCES `item` (`itemID`) ON DELETE CASCADE ON UPDATE CASCADE
+  -- FOREIGN KEY (orderID) REFERENCES `order`(orderID)
+ -- FOREIGN KEY (itemID) REFERENCES item(itemID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `salesAdvisor` (
+  `advisorID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `advisorFirstName` varchar(255) NOT NULL,
+  `advisorLastName` varchar(255) NOT NULL,
+  `storeID` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`advisorID`),
+  CONSTRAINT `salesAdvisor_storeID` FOREIGN KEY (`storeID`) REFERENCES `store` (`storeID`) ON DELETE CASCADE ON UPDATE CASCADE
+ -- FOREIGN KEY (storeID) REFERENCES store(storeID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `order_advisor` (
+  `orderID` int(10) unsigned NOT NULL,
+  `advisorID` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`orderID`, `advisorID`),
+  CONSTRAINT `order_advisor_orderId` FOREIGN KEY (`orderID`) REFERENCES `orders` (`orderID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `order_advisor_advisorID` FOREIGN KEY (`advisorID`) REFERENCES `salesAdvisor` (`advisorID`) ON DELETE CASCADE ON UPDATE CASCADE
+  -- FOREIGN KEY (orderID) REFERENCES `order`(orderID),
+  -- FOREIGN KEY (advisorID) REFERENCES salesAdvisor(advisorID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
```

Comment:
I need to check this, would you provide the ERD for it in the meanwhile as I can't find it here

Reply:
i added directly diagram file but let me add as a screenshot